config/locales/odp.yml

```diff
@@ -53,8 +53,8 @@ en:
       - ['pa.part17', 'Prior Approval - Coal mining']
       - ['pa.part17.classB', 'Prior Approval - Other developments ancillary to mining operations']
       - ['pa.part17.classC', 'Prior Approval - Developments for maintenance or safety']
-      - ['pa.part17.classG', 'Prior Approval - Coal mining development by the Coal Authority for maintenance or safety']
-      - ['pa.part18.classA', 'Prior Approval - Specific Acts of Parliament or Local Orders']
+      - ['pa.part17.classG', 'Prior Approval - Coal mining development by the Coal Authority for maintence or safety']
```

Comment:
Has this misspelling been introduced by them or by us?

Reply:
```suggestion
      - ['pa.part17.classG', 'Prior Approval - Coal mining development by the Coal Authority for maintenance or safety']
```

Reply:
I've basically copy and paste, I saw them as well

Reply:
It looks like they've corrected their spelling errors in `main`. I think we should adopt the corrected spellings; it's not going to make any sense to temporarily have a misspelling in our code.

Particularly in this case where it's only in the description, and not in the identifier itself.

Reply:
yes, that's why I am very keen to improve the process for things like that. so we can avoid human errors